### ISSUE TITLE
fit_generator: same data, different loss

### ISSUE THREAD

**Issue by dzubape** [keras-team/keras#12343]
Hi! I have strange behavior, trying make transfer learning on MobileNetV2 in Keras. I use keras.utils.Sequence class-inheritant as data-generators. One for train, one for validation. But while train loss decreases, validation loss increases since training start. Even if I use single generator for training and for validation (absolutely same data).
My model: https://gist.github.com/dzubape/efeefa77fdb1901d8f99c201b54381df
It seems, that validation loss could be different to training loss, but not like that: 2.5 vs 0.01
Where is my way of thinking wrong?)

**Comment by dzubape** [comment]
I got same behavior without Dropout

**Comment by taobiaoli** [comment]
You should check your datasets，Maybe it's a label problem, or data normalization problem.

**Comment by changken1** [comment]
I am having the same problem. please help francois!
